micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/ExecutorServiceMetrics.java

```diff
@@ -385,22 +387,38 @@ private void monitor(MeterRegistry registry, ForkJoinPool fj) {
             .description("Estimate of the total number of tasks stolen from "
                     + "one thread's work queue by another. The reported value "
                     + "underestimates the actual total number of steals when the pool " + "is not quiescent")
+            .baseUnit(BaseUnits.TASKS)
             .register(registry);
 
         Gauge.builder(metricPrefix + "executor.queued", fj, ForkJoinPool::getQueuedTaskCount)
             .tags(tags)
             .description("An estimate of the total number of tasks currently held in queues by worker threads")
+            .baseUnit(BaseUnits.TASKS)
             .register(registry);
 
         Gauge.builder(metricPrefix + "executor.active", fj, ForkJoinPool::getActiveThreadCount)
             .tags(tags)
             .description("An estimate of the number of threads that are currently stealing or executing tasks")
+            .baseUnit(BaseUnits.THREADS)
```

Comment:
Adding units to existing metrics may be a breaking change depending on the backend. At the least, let's keep that change separate from this PR so one doesn't block the other.